### DISCUSSION TITLE
How to add projects I have already starred to my list?

### DISCUSSION THREAD

**Discussion by effati78** [community/community#8859] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello friends,
I have starred a number of Vue projects in the past and now I want to add them to the Vue list I created. How do I do this?

**Comment by brianlovin** [comment, marked as answer]
Right now the best way is to go to your profile, navigate to the Stars tab (https://github.com/effati78?tab=stars) and click the dropdown button attached to the star button for each repository. We are exploring better ways to bulk-edit a list, but for now it's only possible one at a time.

**Reply by effati78** [reply]
Thank you for your answer 🤍
